### PR TITLE
#27: Update docs with new JavaFX plugin

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: gradle.html [hunk 1]
```diff
@@ -1,70 +1,25 @@
 <h2>Run HelloWorld using Gradle</h2>
 
 <p>
-    Similar to Maven, we can declare JavaFX jars as dependency in the <kbd>build.gradle</kbd> file.
-    However, for Gradle we need to find and specify the platform/OS as classifier. This requires a
-    small script:
+    Similar to Maven, we can declare the required JavaFX modules in the <kbd>build.gradle</kbd> file.
+    However, for Gradle we need to specify the JavaFX gradle plugin:
```
Comment (tiainen) [inline gradle.html]: *apply* instead of specify?

File: ide-eclipse.html [hunk 2]
```diff
@@ -398,8 +404,7 @@ <h5>4. Run the project</h5>
 
 <pre class="alert"><code class="alert">
 run {
-    ...
-    if (platform == 'win') {
+    if (System.getProperty("os.name").startsWith("Windows")) {
```
Comment (tiainen) [inline ide-eclipse.html]: Detect windows with the osdetector plugin.

File: gradle.html (hunk 1 above)
Comment (jperedadnr) [reply gradle.html]: Fixed

File: ide-eclipse.html (hunk 2 above)
Comment (jperedadnr) [reply ide-eclipse.html]: Fixed

**Pull request merged**